src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx

```diff
@@ -4,35 +4,36 @@ import styled from 'react-emotion';
 import OrganizationAvatar from 'app/components/avatar/organizationAvatar';
 import SentryTypes from 'app/sentryTypes';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {Organization} from 'app/types';
 
-class SidebarOrgSummary extends React.Component {
-  static propTypes = {
-    organization: SentryTypes.Organization,
-  };
+type Props = {
+  organization: Organization;
+};
 
-  render() {
-    const {organization} = this.props;
-    const projects = organization.projects && organization.projects.length;
-    const extra = [];
+const SidebarOrgSummary: React.FC<Props> = ({organization}) => {
+  const projects = organization.projects && organization.projects.length;
+  const extra: string[] = [];
 
-    if (projects) {
-      extra.push(`${projects} projects`);
-    }
+  if (projects) {
+    // should we think about pluralization here?
+    extra.push(`${projects} projects`);
```

Comment:
We likely should use `tn()` in the locale module to translation and pluralization.

Reply:
used it, thanks 👍